packages/ui/cypress/e2e/app/agents/write/create.js

```diff
@@ -1,21 +1,91 @@
 export default () =>
-  describe('form crud', () => {
-    describe('create', () => {
-      beforeEach(() => {
-        cy.goToRoute('/app/agents?mainColumn=create')
-        cy.get('[data-testid="agentRoot"]', { log: false, timeout: 20000 })
+  describe('create new agent', () => {
+    before(() => {
+      cy.resetDevelopmentSqlDb()
+      cy.visit('/app/agents?mainColumn=create')
+      cy.get('[data-testid="agentRoot"]', { log: false, timeout: 20000 })
+    })
+
+    it('fills in and save agent form', () => {
+      cy.log(
+        'adds agent type, full name, disambiguating description and remarks'
+      )
+      cy.getByElementName('agentType').check('organization', { force: true })
+      cy.getByElementName('fullName').type('agentName')
+      cy.getByElementName('disambiguatingDescription').type(Date.now())
+      cy.getByText('Add agent remarks...').click()
+      cy.getByElementName('remarks').type('some remarks text')
+
+      cy.log('adds name details')
+      cy.getByText('Add name details').click()
+      cy.getByElementName('givenName').type('Tester')
+      cy.getByElementName('familyName').type('Tester')
```

Comment:
generally I think it's good to use the elementName as the test string, e.g. familyName instead of Tester, just so that if some error shows and it says "Tester" it might not be clear which of the four "Tester" fields it refers to :-) (and it means it's less work to come up with test strings)